modules/standard/Spawn.chpl

```diff
@@ -443,9 +480,12 @@ module Spawn {
           }
         }
 
-    if stdin == QIO_FD_PIPE then stdin_pipe = true;
-    if stdout == QIO_FD_PIPE then stdout_pipe = true;
-    if stderr == QIO_FD_PIPE then stderr_pipe = true;
+    if stdin == QIO_FD_PIPE || stdin == QIO_FD_BUFFERED_PIPE then
+      stdin_pipe = true;
+    if stdout == QIO_FD_PIPE || stdout == QIO_FD_BUFFERED_PIPE then
+      stdout_pipe = true;
+    if stderr == QIO_FD_PIPE || stderr == QIO_FD_BUFFERED_PIPE then
```

Comment:
Might be worth making this a function that checks if the channel provided is a pipe of any sort, but it's not a big deal